frontend/src/pages/CourseEditor/SkillsStep/SkillsStep.tsx

```diff
@@ -18,80 +20,92 @@ import {
   SkillsTitleWrapper,
   SkillWrapper,
 } from './styled';
-import { ISkillsStepProps } from '../types';
-import { getPointsArr } from '../utils';
 
 const SkillsStep: FC<ISkillsStepProps> = ({
-  courseData,
   formik,
-  isCourseDataLoading,
   handleChangeTechnology,
+  unGroupedSkills,
+  isCreateCourseMode,
+  isCourseDataLoading,
 }) =>
   isCourseDataLoading ? (
     <Loader type="content" />
   ) : (
     <FormWrapper>
-      <SectionName>{EditorTitles.skillStepTitile}</SectionName>
+      <SectionName>{EditorTitles.skillStepTitle}</SectionName>
       <SkillsTitleWrapper>
         <SkillsText>{EditorTitles.skillDescription}</SkillsText>
       </SkillsTitleWrapper>
       <FieldArray name="technologies">
         {({ remove, push }) => (
           <>
-            {courseData &&
-              courseData.allSkills.length &&
-              formik.values.technologies.map((technology, index) => (
-                <SkillWrapper key={index}>
-                  <>
-                    <InnerWrapper>
-                      <SkillField
-                        select
-                        variant="outlined"
-                        label="Technology"
-                        value={technology._id || ''}
-                        id={`technologies[${index}]._id`}
-                        name={`technologies[${index}]`}
-                        onChange={handleChangeTechnology}
-                        error={Boolean(formik.errors?.technologies?.[index]?.name)}
-                        helperText={formik.errors?.technologies?.[index]?.name}
-                      >
-                        {courseData &&
-                          courseData.allSkills.map((skill) => (
-                            <MenuItem key={skill._id} value={skill._id}>
-                              {skill.name}
+            {formik.values.technologies.map((technology, index) => (
+              <SkillWrapper key={index}>
+                <>
+                  <InnerWrapper>
+                    <SkillField
+                      select
+                      variant="outlined"
+                      label="Technology"
+                      value={isCreateCourseMode ? technology.name : technology._id}
```

Comment:
should we use the name as the `value` if we need to add an `id` to the API request body?
<img width="418" alt="image" src="https://user-images.githubusercontent.com/44285339/173369801-2e961090-a8d7-490a-a743-18dbe1ec3335.png">